src/_P012_LCD.ino

```diff
@@ -264,12 +268,14 @@ String P012_parseTemplate(String& tmpString, byte lineSize) {
   String result            = parseTemplate_padded(tmpString, lineSize);
   const char degree[3]     = { 0xc2, 0xb0, 0 }; // Unicode degree symbol
   const char degree_lcd[2] = { 0xdf, 0 };       // P012_LCD degree symbol
+
   result.replace(degree, degree_lcd);
 
   char unicodePrefix = 0xc4;
-#ifdef USES_P012_POLISH_CHARS
-  if (result.indexOf(unicodePrefix) != -1) {
 
+# ifdef USES_P012_POLISH_CHARS
+
+  if (result.indexOf(unicodePrefix) != -1) {
```

Comment:
I pressed the Format Source hotkey, so this, and source code below, was (also) reformatted to Uncrustify settings 😉